### ISSUE TITLE
Spotlight preference fades out the site editor frame

### ISSUE THREAD

**Issue by draganescu** [WordPress/gutenberg#51167]
### Description

When the Spotlight preference is set to on, then the preview of the template or page in the site editor is all faded out as no block is selected.

### Step-by-step reproduction instructions

1. Have a block theme active
2. Go to Appearance > Editor
3. Click on the preview
4. From the top right dot menu set Spotlight mode to on
5. Click the site hub (site logo top left corner)
6. Notice the preview is not faded

### Screenshots, screen recording, code snippet

<img width="1192" alt="Screenshot 2023-06-01 at 14 51 19" src="https://github.com/WordPress/gutenberg/assets/107534/a2c778bf-2063-4c0a-bd04-57fbae35b91b">

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by faisal-alvi** [comment]
@draganescu I have tested and I can reproduce the issue. I noticed the `.is-focus-mode` class is removed when the sidebar is opened.

![gb-51167](https://github.com/WordPress/gutenberg/assets/25176325/47ccf5a5-1bf1-472d-b62f-fd133b7196d8)

However, this seems to be an intentional behavior. As far as the user wants to access the sidebar, it means the user no longer requires focus mode enabled and it is pointless to keep it faded.
